### PR TITLE
Update 5-letter-finder.md with correct Example

### PR DESCRIPTION
**Pull request by pboos**
Fix correct answer ;-)

Btw. There are even more answers. Considering that you can as well just go the other way. See all answers here:

Startposition and then the movements taken in x and y direction (0,0 being the left top)
1,0 -> [-1,0], [0,1], [0,1], [1,0], [0,-1], [1,1], [1,-1], [0,-1], [-1,0], [0,1]
1,0 -> [-1,0], [0,1], [0,1], [1,0], [0,-1], [1,1], [1,-1], [-1,0], [0,-1], [1,0]
1,2 -> [-1,0], [0,-1], [0,-1], [1,0], [0,1], [1,1], [1,-1], [0,-1], [-1,0], [0,1]
1,2 -> [-1,0], [0,-1], [0,-1], [1,0], [0,1], [1,1], [1,-1], [-1,0], [0,-1], [1,0]
2,1 -> [0,-1], [1,0], [0,1], [-1,1], [-1,-1], [0,-1], [-1,0], [0,1], [0,1], [1,0]
2,1 -> [0,-1], [1,0], [0,1], [-1,1], [-1,-1], [0,1], [-1,0], [0,-1], [0,-1], [1,0]
3,0 -> [-1,0], [0,1], [1,0], [-1,1], [-1,-1], [0,-1], [-1,0], [0,1], [0,1], [1,0]
3,0 -> [-1,0], [0,1], [1,0], [-1,1], [-1,-1], [0,1], [-1,0], [0,-1], [0,-1], [1,0]

### CODE REVIEW

**Comment by yanarchy** [comment]
Yay THANK YOU!! Omg, this is the second pull request someone has made EVER.

File: meetup-5/5-letter-finder.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ The longest length (an integer).
 
 ## Example
 
-Given the above example, the answer would be 10.
+Given the above example, the answer would be 11.
 
-Reasoning: Starting from the `a` in the top-left corner...
+Reasoning: Starting from the `b` in the second column in the top row...
```
Comment (yanarchy) [inline meetup-5/5-letter-finder.md]: I don't understand this change. The instruction says that it has to start in the top-left corner.

You could add an advanced portion?

Comment (pboos) [reply meetup-5/5-letter-finder.md]: Ups, I guess I should read the instructions better. In that case never mind the pull request.

But starting up there makes it too easy ;-). Guess I wrote too much code then :D.

Comment (yanarchy) [reply meetup-5/5-letter-finder.md]: You can add in an advanced section for it! 🌸

Comment (pboos) [reply meetup-5/5-letter-finder.md]: Nono, totally fine. I think having it only from the left top makes it simpler. The hard part of the algorithm is not the part from having it able to start anywhere. It only adds one simple loop (on bruteforce at least. not sure if that would make a bigger difference for better algorithms).

So leaving it like it is is fine I think.

**Pull request closed without merging**